packages/mds-stream/kafka/stream-consumer.ts

```diff
@@ -42,8 +84,12 @@ const createStreamConsumer = async (
 
     const kafka = new Kafka({ clientId, brokers })
     const consumer = kafka.consumer({ groupId })
+
     await consumer.connect()
     await Promise.all(asArray(topics).map(topic => consumer.subscribe({ topic, fromBeginning })))
+
+    registerLivelinessHandling(consumer, healthStatus)
```

Comment:
I really like that this matures the handling of all the consumer lifecycle events, even if not all of them result in a health-monitor status change.